### PR TITLE
New responsive motion gallery with touch gesture support

### PR DESCRIPTION
**Pull request by Erftralle**
This pull request introduces an alternative for the thumbnail scroller resp. motion gallery used in JoomGallery's detail view.

It is responsive and does support touch gestures for mobile devices. Details can be found [here](http://manos.malihu.gr/jquery-thumbnail-scroller/) and the source can be found on GitHub [here](https://github.com/malihu/thumbnail-scroller).

If testing it you should not note any big differences to the current motion gallery, but a small one there is. The new thumbnail scroller does not center the actual shown image on page load, it shows it as far as possible to the left.
I opened an issue at the [project's github page](https://github.com/malihu/thumbnail-scroller/issues/4), lets see what happens.

### CODE REVIEW

**Comment by Chraneco** [comment]
Awesome, this works great on my phone!

A few small things that I discovered:
- In this demo the image with the white border can be centered: http://manos.malihu.gr/repository/jquery-thumbnail-scroller/demo/examples/scrollTo_demo.html. It's using something like ```$("#white").position().left+($("#white").width()/2)-($(el).find(".mTSWrapper").width()/2);``` to accomplish that. This way we would not have any drawbacks compared to the old version anymore.
- When scrolling completely to the right or to the left, that last image does not directly move in completely. For example, if you move the cursor to the right and wait until the motion bar stopped, the last thumbnail moves in only half way and you have to fiddle with the mouse on top of it in order to move the rest in. It seems that something like ```hover-10``` instead of ```hover-precise``` works better in that regard.
- The width of the bar does not seem to be calculated correctly. The right border of the right-most image is always hidden.
- During page load the thumbnails appear stacked with a scroll bar on the right. A loading indicator like in this demo http://manos.malihu.gr/repository/jquery-thumbnail-scroller/demo/examples/show_when_ready_example.html would be better in my opinion even if the styling does not quite fit, but maybe the loading icon can be replaced.

File: components/com_joomgallery/views/detail/view.html.php [hunk 1]
```diff
@@ -626,29 +626,9 @@ function joom_hover() {
...
-              . "  <style type=\"text/css\">\n"
-              . "     #motioncontainer {\n"
-              . "       width:expression(Math.min(this.offsetWidth, maxwidth)+'px');\n"
-              . "     }\n"
-              . "  </style>\n"
-              . "  <![endif]-->"
-              . "\n"
-              . "  <!-- End Conditional Style -->";
-      $this->_doc->addCustomTag($custom);
+      JHtml::_('bootstrap.framework');
```
Comment (Chraneco) [inline components/com_joomgallery/views/detail/view.html.php]: Is bootstrap necessary here?
```jquery.framework``` should be enough.

Comment (Erftralle) [inline components/com_joomgallery/views/detail/view.html.php]: You are right, thank you.

**Comment by Erftralle** [comment]
Thank you for your detailed and constructive feedback.

With the latest changes

- the thumbnail of the actual detail image displayed is now centered (if possible)
- scroller type is changed to ```hover-20```, which I agree is much better than ```hover-precise```
- I removed the negative margin from ```jg_minipic``` again, which caused the right border of the right-most image not to be shown
- the problems on page load is solved too, I did not use any spinners and transition effects to keep it simple

**Comment by Chraneco** [comment]
Great, thank you very much!

**Pull request merged**